anna-sala/challenges/pokeapi/list/list.js

```diff
@@ -0,0 +1,50 @@
+const list = document.querySelector('.pokemons__list');
+const oldLimit = new URLSearchParams(window.location.search).get('limit');
+let oldOffset = new URLSearchParams(window.location.search).get('offset');
+const pokemonPage = new URLSearchParams(window.location.search).get('pokemonName');
+
+function printOnListPage(tag, text, idName, url) {
+  const listItem = document.createElement('li');
+  const content = document.createElement(tag);
+  listItem.setAttribute('class', idName);
+  content.innerHTML = text;
+  content.href = url;
+  listItem.appendChild(content);
+  list.appendChild(listItem);
+}
+
+const removeAllChildNodes = (parent) => {
+  while (parent.firstChild) {
```

Comment:
When does this while stops ?

Reply:
The while stops when there are no more children for that parent.